### PR TITLE
test(clone): finish clone gameObject redo/undo test

### PR DESCRIPTION
**Pull request by AmyOrz**

### CODE REVIEW

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_addGameObject_test.js.snap [hunk 1]
```diff
@@ -0,0 +1,794 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`redo_undo: add gameObject prepare first step: set currentSceneTreeNode test redo operate test redo one step undo step which from second to zero, redo step which from zero to first 1`] = `
```
Comment (yyc-git) [inline test/integration/redo_undo/gameObject/__snapshots__/redo_undo_addGameObject_test.js.snap]:
```suggestion
ddd
```

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap [hunk 2]
```diff
@@ -0,0 +1,518 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`redo_undo: dispose gameObject prepare first step: set currentSceneTreeNode test operate disposeGameObject(because the set currentSceneTreeNode operation is redoUndoable, so need execute redo/undo operation twice for dispose one gameObject) test redo operate test redo one step undo step which from second to zero,redo step which from zero to first 1`] = `
+<article
```
Comment (yyc-git) [inline test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap]: TODO need move func out here

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+      <li>
+        <div
+          className="item-triangle"
+          onMouseDown={[Function]}
+        >
+          <img
+            src="./public/img/down.png"
+          />
+        </div>
+        <div
```
Comment (yyc-git) [inline test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap]: TODO change a

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+          onDragOver={[Function]}
+          onDragStart={[Function]}
+          onDrop={[Function]}
+          onMouseDown={[Function]}
+          style={Object {}}
+        >
+          Scene
+        </div>
+      </li>
+      <ul
```
Comment (yyc-git) [inline test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap]: TODO change b

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_addGameObject_test.js.snap (hunk 1 above)
Comment (AmyOrz) [reply test/integration/redo_undo/gameObject/__snapshots__/redo_undo_addGameObject_test.js.snap]: not change

File: test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap [hunk 5]
```diff
@@ -0,0 +1,518 @@
...
+        >
+          <img
+            src="./public/img/down.png"
+          />
+        </div>
+        <div
+          className="draggable-container"
+          draggable={true}
+          onDragEnd={[Function]}
+          onDragEnter={[Function]}
```
Comment (AmyOrz) [inline test/integration/redo_undo/gameObject/__snapshots__/redo_undo_disposeGameObject_test.js.snap]: aaaaa
